websites/D/Discord FR/presence.ts

```diff
@@ -0,0 +1,191 @@
+    }
+    else if (document.location.pathname.includes("/recrutement/")) {
+      presenceData.details = "Consulte le formulaire";
+      presenceData.state = "de recrutement";
+      presenceData.largeImageKey = "recrutement";
+      presenceData.smallImageKey = "rooster";
+      presenceData.smallImageText = "discord.fr/recrutement";
+    }
+    else if (document.location.pathname.includes("/partenariat/")) {
+      presenceData.details = "Consulte le formulaire";
+      presenceData.state = "de partenariat";
+      presenceData.largeImageKey = "partenariat";
+      presenceData.smallImageKey = "rooster";
+      presenceData.smallImageText = "discord.fr/partenariat";
+    }
+    else if (document.location.pathname.includes("/unban/")) {
+      presenceData.details = "Consulte le formulaire";
+      presenceData.state = "d'appel";
+      presenceData.largeImageKey = "unban";
+      presenceData.smallImageKey = "rooster";
+      presenceData.smallImageText = "discord.fr/unban";
+    }
+    else {
+      presenceData.details = "Erreur lors du chargement";
+      presenceData.state = "discord.fr";
+    }
+
+  }
+
+  else if (document.location.hostname == "i.discord.fr") {
+      presenceData.details = "Consulte une image";
+      presenceData.state = "i.discord.fr";
+      presenceData.largeImageKey = "image";
+      presenceData.smallImageKey = "dfr";
+      presenceData.smallImageText = "i.discord.fr";
+    }
+  else if (document.location.hostname == "status.discord.fr") {
+      presenceData.details = "Consulte l'état des services";
+      presenceData.state = "status.discord.fr";
+      presenceData.largeImageKey = "status";
+      presenceData.smallImageKey = "dfr";
+      presenceData.smallImageText = "status.discord.fr";
+    }
+
+  else if (document.location.hostname == "staff.discord.fr") {
+    presenceData.details = "Consulte la";
+    presenceData.state = "documentation staff";
+    presenceData.largeImageKey = "staff";
+    presenceData.smallImageKey = "rooster";
+    presenceData.smallImageText = "staff.discord.fr";
+    }
+
+  else if (document.location.hostname == "support.discord.fr") {
+    if (document.location.pathname == "/") {
+      presenceData.details = "Team Support";
+      presenceData.state = "via ModMail";
+      presenceData.largeImageKey = "support";
+      presenceData.smallImageKey = "dfr";
+      presenceData.smallImageText = "support.discord.fr";
+    }
+    else if (document.location.pathname.includes("/logs/")) {
+      presenceData.details = "Consulte une archive";
+      presenceData.state = "d'un ticket support";
+      presenceData.largeImageKey = "logs";
+      presenceData.smallImageKey = "dfr";
+      presenceData.smallImageText = ""; 
+    }
+    else {
+      presenceData.details = "Erreur lors du chargement";
+      presenceData.state = "support.discord.fr";
+    }
+
+  }
+  else {
+    presence.setActivity();
+    presence.setTrayTitle();
+  }
+
+  updatePresenceData();
```

Comment:
still why ;-; please use ``presence.on("UpdateData", async () => { });`` pls not different function bruh.